ui/web/src/Localization/Es.elm

```diff
@@ -0,0 +1,150 @@
+module Localization.De exposing (esLocalization, esTranslations)
```

Comment:
```suggestion
module Localization.Es exposing (esLocalization, esTranslations)
```

Reply:
Yeah, I was already making the change. Thanks!